src/main/java/yablunovskyi/bookstore/controller/BookController.java

```diff
@@ -33,6 +34,7 @@ public class BookController {
     @ResponseStatus(HttpStatus.OK)
```

Comment:
just FYI

`OK` is the default response status, so you can omit it

remember about this in future :)

Reply:
Nice, thanks!